gmso/external/convert_mol2.py

```diff
@@ -0,0 +1,175 @@
+"""Convert to and from a TRIPOS mol2 file."""
+# TODO add sources of mol2 files
+import os
+import warnings
+
+import unyt as u
+
+from gmso import Atom, Bond, Box, Topology
+from gmso.core.element import element_by_name, element_by_symbol
+
+
+def from_mol2(
+    filename, site_type="Atom"
+):  # TODO add flags for information to return
+    """Read in a TRIPOS mol2 file format into a gmso topology object.
+
+    Creates a Topology from a mol2 file structure. This will read in the
+    topological structure (sites, angles, and box) information into gmso.
+    Note that parameterized information can be found in these objects, but
+    will not be converted to the Topology.
+
+    Parameters
+    ----------
+    filename : string
+        path to the file where the mol2 file is stored.
+    site_type : string, optional, default='Atom'
```

Comment:
consider listing the options ('Atom', 'lj') in this line, as I think it makes the options quickly clearer for the reader.

Reply:
Addressed!

Reply:
This line is a little confusing to me. Maybe something like this is better?

site_type : string ('atom' or 'lj'), default='atom'